### PR TITLE
Get comments

### PR DESCRIPTION
**Pull request by LewPearce**
added an expandable comments section to each individual review

### CODE REVIEW

File: src/Components/Comments.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import { useEffect, useState } from "react";
+import { getComments } from "../axios/api";
+import { useParams } from "react-router-dom";
+
+export const Comments = ({ setComments, review_id, comments }) => {
+  useEffect(() => {
+    getComments(review_id).then((review) => {
```
Comment (JVal-NC) [inline src/Components/Comments.js:10]: Could we have some loading feature for this so a user knows why content may not be showing immediately.

Maybe also consider if no comments exist?

File: src/Components/Comments.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+      setComments(review.comments);
+    });
+  }, [setComments, review_id]);
+
+  return (
+    <>
+      <ul className="comment__list">
+        <li></li>
```
Comment (JVal-NC) [inline src/Components/Comments.js]: Unnecessary list tags?

File: src/Components/Review.js [hunk 3]
```diff
@@ -48,9 +60,18 @@ export const Review = ({ reviewList, setReviewList }) => {
...
+            Show Comments
+          </button>
+          {wantComments ? (
+            <Comments
+              setComments={setComments}
+              review_id={review_id}
+              comments={comments}
+            />
+          ) : (
+            <div></div>
```
Comment (JVal-NC) [inline src/Components/Review.js]: Got a feeling these div tags arent needed

**Review by JVal-NC — COMMENTED** [review]
Strong start, some minor things to add in the comments. Good work.

File: src/Components/Review.js (hunk 3 above)
Comment (LewPearce) [reply src/Components/Review.js]: if I remove those div tags the page doesn't load, obviously they're just there to render the absence of a comment list, is there a better approach?

**Pull request merged**